src/commands/lambda/flare.ts

```diff
@@ -229,6 +242,62 @@ export class LambdaFlareCommand extends Command {
   }
 }
 
+/**
+ * Obfuscate the environment variables in a Lambda function configuration
+ * @param config
+ */
+export const obfuscateConfig = (config: FunctionConfiguration) => {
+  const environmentVariables = config.Environment?.Variables
+  if (!environmentVariables) {
+    return config
+  }
+
+  const obfuscatedEnvironmentVariables: {[key: string]: string} = {}
+  for (const [key, value] of Object.entries(environmentVariables)) {
+    obfuscatedEnvironmentVariables[key] = value
+    if (!NON_OBFUSCATED_ENV_VARS.has(key)) {
```

Comment:
This reads as `if not not_array has key` could we improve the naming to make it simpler?

Something that could be read as `if key in array skip`, maybe even just improving the name can lead to a change.

Something like `SKIP_MASKING_ENV_VARS`, whatever you think its best
```suggestion
    if (SKIP_MASKING_ENV_VARS.has(key)) {
        continue
    }
    // mask here
```

Reply:
Good suggestion! Changed